CONTRIBUTING.md

```diff
@@ -295,71 +295,43 @@ Once substantial time has passed or significant changes have been made to `pkgne
+Github Actions workflows will now: 
+- test your PR against latest `ubuntu` and `macos` R versions (like all PRs)
+- check that your version numbering is correct (and you didn't leave the trailing `.9999`) 
+- test your PR against R development version ("devel") on the previous OS's as well as Microsoft (mirroring CRAN checks for package updates)
 
-```
-Rscript -e "roxygen2::roxygenize()"
-Rscript -e "install.packages('pkgdown', repos = 'cran.rstudio.com')"
-Rscript -e "pkgdown::build_site()"
-```
-
-`pkgdown` is changing pretty rapidly, so it's important to pull the latest from CRAN before building the site.
-
-Check in whichever of the files generated by `pkgdown` seem relevant to you. Anything that shows up as "modified" when you run `git status` should be checked in to the PR branch. New files ("not tracked by git") should be considered on a case-by-case basis.
-
-### Test on latest development version of R (i.e. "R-Devel") 
-
-This is a CRAN requirement, and yes, it's a pain.  Luckily, Dirk Eddelbuettel and friends maintain docker images to make this process (a little) easier.  Specifically, the [rocker/drd](https://hub.docker.com/r/rocker/drd) image from [the Rocker Project](https://www.rocker-project.org/images/) provides a great jumping off point.  It's a lighter r-devel image.  Note, this image is rebuilt every few days, so if you _absolutely need_ to have _bleeding edge_ r-devel and have a extra hour to spare, follow [this docker process](https://alexandereckert.com/post/testing-r-packages-with-latest-r-devel/) from Alexander Eckert. 
+If some checks fail, correct and push again to the release branch.  
 
-1. Navigate to the base directory of the package. 
-2. Build the r-devel pkgnet docker image and run CRAN checks via these commands: 
-```
-# On Windows
-docker pull rocker/drd
-docker build -f release_testing\dockerfile -t pkgnet_rdevel . 
-docker run --rm -ti -v %cd%:/RPackage pkgnet_rdevel ./r_devel_check.sh
-
-# On Mac
-docker pull rocker/drd
-docker build -f release_testing/dockerfile -t pkgnet_rdevel . 
-docker run --rm -ti -v $(pwd):/RPackage pkgnet_rdevel ./r_devel_check.sh
-```
-3. Fix your errors and rerun container (second line) until clean CRAN level checks.
-4. Once your satisfied with your results, you're done with _linux_ checks.  
-5. To test on windows r-devel (as CRAN will do), take the final package tarball and upload it to this webpage: https://win-builder.r-project.org/upload.aspx
-6. Grab a coffee and wait for an email (about 10 minutes later but varies)
-7. Fix your errors and reupload until good. 
+Once checks all pass, _Don't merge the PR quite yet!_  Follow the steps below to [submit to CRAN](https://cran.r-project.org/submit.html) which is still a minor manual process.
 
 ### Submit to CRAN
 
-Go to https://cran.r-project.org/submit.html, take the tarball that passed the gauntlet and submit it! 
+1. Build the tarball locally on the current release branch: `R CMD Build .`
```

Comment:
Wondering if it makes sense to do this in the CI. Then you could just download the tarball if the tests pass. 

https://github.com/actions/upload-artifact

Reply:
That's a great call out.  TIL.  Assuming the uploaded artifact is just then available and downloadable from the run GUI/CLI?

Reply:
Yeah, so here's an example CI run where I'm testing a tool that generates some images, and I wanted to persist those images so I could manually look at them if I wanted to.

You can see the download links if you scroll down to the bottom to "Artifacts": https://github.com/drivendataorg/erdantic/actions/runs/8677103132

Here's the `actions/upload-artifact` step: https://github.com/drivendataorg/erdantic/blob/c687b03d3647b619788e7396043ab927afbe057d/.github/workflows/tests.yml#L65-L70